laborator/content/structuri-vectori-siruri/3-4-5-getters-setters/getter_setter_printf.asm

```diff
@@ -29,31 +29,55 @@ global main
 get_int:
     ; TODO --- return the int value from struct
```

Comment:
Please add 1 patch per logical change. So, this Pull request will have 3 patches:
* 1 to modify the lab number
* 1 to modify the skeleton for getter/setters
* 1 to modify the find_substring skel.